src/app/components/tree/tree.component.ts

```diff
@@ -39,6 +39,7 @@ import { FileNode } from '../../structures/file-node';
 export class TreeComponent implements AfterContentInit, OnDestroy {
   @Input() treeData: TreeNode;
   @Input() style: any;
+  @Input() noResultMsg: String;
```

Comment:
We should probably use the primitive type 'string' unless you have a good reason not to